### DISCUSSION TITLE
R1 LED1 suggestions?

### DISCUSSION THREAD

**Discussion by RuthlessNate** [pmgducati/Blue-Retro-AIO-Units#36] [Q&A] [answered]
Hey guys, I'm pretty new to this kind of thing. I've started putting together the whole thing. I realized there are pads for an LED on the main board that will be visible out the top of the 3D printed case. Any suggestions on what LED I should buy and how it lines up with the pads? I see one pad on the 3v3 circuit and the two on the left connecting to the I017 via. 

Thanks

**Comment by pmgducati** [comment, marked as answer]
Hi,
You can find the Digikey part number in the BOM.  I just updated it, the spreadsheet had all the correct info but the print area was incorrect which cut off the main PCB BOM.

**Reply by RuthlessNate** [reply]
Thanks!
